tests/plugins/balloontoolbar/_helpers/default.js

```diff
@@ -16,3 +16,16 @@ function convertRgbaToRgb( input ) {
 	var re = /^rgba\(\s*\d+\s*,\s*\d+\s*,\s*\d+\s*,\s*\d+\s*\)$/gi;
 	return re.test( input ) ? input.replace( /,\s*?\d+?\s*?\)$/, ')' ).replace( 'rgba(', 'rgb(' ) : input;
 }
+
+function ignoreUnsupportedEnvironment( testSuite, check ) {
+	var isSupported = CKEDITOR.env.ie && CKEDITOR.env.version > 8;
```

Comment:
Current condition also ignore test on all non-ie browser like Chrome, FF, Safari, etc.
You can:
- reverse the condition to check unsupported browser and then correct further condition
- or extend this condition to accept all non-ie browsers